exercise_03/generateRandomNumberInRange.js

```diff
@@ -3,3 +3,16 @@
  * input: number (limit)
  * output: number
  */
+
+/**
+ * 
+ * @param {number} limit 
+ * @returns {number}
+ */
+const generateRandomNumberInRange = function (limit){
+
+    return Math.random() * limit | 0;
```

Comment:
Why are you using this logical operator in this case?

- [ ] 🚨 Explain this expression.

Reply:
Becouse the expression Math.random() * limit | 0 ensures that the result is an integer between 0 and limit - 1.